src/components/diagrams/singleLineDiagram/single-line-diagram-pane.js

```diff
@@ -205,7 +205,7 @@ export function SingleLineDiagramPane({
                 ?.ref?.current?.reloadSvg();
         } else
             viewsRef.current.forEach((sld) => {
-                if (sld.svgUrl.indexOf(currentNodeRef.current?.id) !== -1) {
+                if (sld.svgUrl && sld.svgUrl?.indexOf(currentNodeRef.current?.id) !== -1) {
```

Comment:
You then do not need the first `?.` because at this point sld.svgUrl is not undefined nor null.

Reply:
ok. done.